README.md

```diff
@@ -30,6 +30,13 @@ find . -name "*_shpec.sh" | entr shpec
 [shpec's own tests](https://github.com/rylnd/shpec/tree/master/shpec/shpec_shpec.sh)
 are a great place to start. More detailed examples to come.
 
+### Mocking
```

Comment:
Can we move this section after 'Matchers' and call it "Command Stubbing" ? We should be consistent with the terminology.

Also, a [link to the example tests](https://github.com/rylnd/shpec/blob/master/shpec/shpec_shpec.sh#L71-L84) would be useful.